smart-contracts/osmosis/contracts/cl-vault/src/vault/deposit.rs

```diff
@@ -61,10 +60,9 @@ pub(crate) fn execute_any_deposit(
         .position
         .ok_or(ContractError::MissingPosition {})?;
 
-    // get the amount of funds we can deposit from this ratio
-    // let (deposit_amount_in_ratio, swappable_amount): ((Uint128, Uint128), (Uint128, Uint128)) =
     let deposit_info = get_depositable_tokens(&deps.branch(), &info.funds, &pool_config)?;
 
+    // If we have no refunds let's proceed for the deposit
```

Comment:
idk, because of the early return. so to leave a clear feedback that this workflow ends here if that condition is met. i agree that the below code is explicative enough tho so we could avoid.